x/mongo/driver/connstring/connstring_test.go

```diff
@@ -102,6 +102,17 @@ func TestConnect(t *testing.T) {
 		{s: "connect=AUTOMATIC", expected: connstring.AutoConnect},
 		{s: "connect=direct", expected: connstring.SingleConnect},
 		{s: "connect=blah", err: true},
```

Comment:
Maybe add same tests for directConnction?

    directConnection=TRUE
    directConnection=FALSE
    directConnection=blah